### PR TITLE
i18n: add new config ja/cn/ko translation

### PR DESCRIPTION
**Pull request by MaikoTan**
@Echoring Please review.

### CODE REVIEW

File: ui/raidboss/raidboss_config.js [hunk 1]
```diff
@@ -995,6 +995,8 @@ const templateOptions = {
       id: 'Coverage',
       name: {
         en: 'Supported content (latest version)',
+        ja: 'コンテンツ完成度一覧 (最新バージョン)',
+        cn: '副本覆盖度 (最新版本)',
```
Comment (Echoring) [inline ui/raidboss/raidboss_config.js:999]:
```suggestion
        cn: '支持副本一览 (最新版本)',
```

File: ui/raidboss/raidboss_config.js [hunk 2]
```diff
@@ -995,6 +995,8 @@ const templateOptions = {
       id: 'Coverage',
       name: {
         en: 'Supported content (latest version)',
+        ja: '対応コンテンツ一覧 (最新バージョン)',
+        cn: '支持副本一览 (最新版本)',
```
Comment (Jaehyuk-Lee) [inline ui/raidboss/raidboss_config.js:999]:
```suggestion
        cn: '支持副本一览 (最新版本)',
        ko: '지원하는 컨텐츠 (릴리즈버전보다 최신)',
```

Comment (MaikoTan) [reply ui/raidboss/raidboss_config.js:999]: You quoted "Newer than release"?
I wonder if we should add in ja/cn too?

Comment (Jaehyuk-Lee) [reply ui/raidboss/raidboss_config.js:999]: I don't know... It's very hard to describe this "commit-release" step of publishing to non-developers. I just tried to notice that this list may contain that are not released yet.

Comment (MaikoTan) [reply ui/raidboss/raidboss_config.js:999]: Oh yeah. And especially for CN/KO server which is outdated compare to the intl server. So there might be some "un-implemented content" included.
I think I would keep the Japanese translation remain, and ping @Echoring for suggestion again.

Comment (Echoring) [reply ui/raidboss/raidboss_config.js:999]: hmmm...... maybe we can use a more common word like `Developing version included` or `Beta version included`？

Comment (Jaehyuk-Lee) [reply ui/raidboss/raidboss_config.js:999]: `Developing version included` looks good.

Comment (Jaehyuk-Lee) [reply ui/raidboss/raidboss_config.js:999]: or let's use `<br>` and describe it long like
```
Supported content
(not-yet released contents included)
```

Comment (Jaehyuk-Lee) [reply ui/raidboss/raidboss_config.js:999]: Or maybe we could add some description on the `coverage.html` page

Comment (MaikoTan) [reply ui/raidboss/raidboss_config.js:999]: Hmm, I would agree to add a description on the HTML page.
As for the CN translation, I think `not-yet released contents included` is good, I can abbreviate it into "含未发布更新" which has only 6 characters.

Comment (Jaehyuk-Lee) [reply ui/raidboss/raidboss_config.js]: Imo, most Korean users are aware that the international version has more contents updated, they'll understand what English named ones are like "The Tower at Paradigm's Breach" in Korean page. I don't think Korean need something like that.

Comment (MaikoTan) [reply ui/raidboss/raidboss_config.js]: Well, I mainly play in JP, so I'm not very clear that if most CN players are all the same.
If there is a description, I would translate it as soon as possible. ^_^

**Pull request merged**